extension/package.json

```diff
@@ -409,39 +409,12 @@
         "category": "DVC",
         "icon": "$(source-control)"
       },
-      {
-        "title": "%command.views.experimentsTree.queueExperiment%",
-        "command": "dvc.views.experimentsTree.queueExperiment",
-        "category": "DVC",
-        "icon": {
-          "dark": "resources/dark/queue-experiment.svg",
-          "light": "resources/light/queue-experiment.svg"
-        }
-      },
       {
         "title": "%command.views.experimentsTree.removeExperiment%",
         "command": "dvc.views.experimentsTree.removeExperiment",
         "category": "DVC",
         "icon": "$(trash)"
       },
-      {
-        "title": "%command.views.experimentsTree.runExperiment%",
-        "command": "dvc.views.experimentsTree.runExperiment",
-        "category": "DVC",
-        "icon": "$(play)"
-      },
-      {
-        "title": "%command.views.experimentsTree.resumeCheckpointExperiment%",
-        "command": "dvc.views.experimentsTree.resumeCheckpointExperiment",
-        "category": "DVC",
-        "icon": "$(debug-continue-small)"
-      },
-      {
-        "title": "%command.views.experimentsTree.resetAndRunCheckpointExperiment%",
-        "command": "dvc.views.experimentsTree.resetAndRunCheckpointExperiment",
-        "category": "DVC",
-        "icon": "$(play)"
-      },
```

Comment:
[F] For these 4 commands we need to pause updates. Which means that have to go through the `WorkspaceExperiments` path

Reply:
One downside of this is that the context menu titles blow out:

![image](https://user-images.githubusercontent.com/37993418/176355881-b9c9285f-e7c7-4e8d-93fc-4d0abebdcc8b.png)

But an upside is that the correct toasts are always shown if there is output or an error that the user needs to see.